### ISSUE TITLE
Extensive GPU Usage for Manipulation

### ISSUE THREAD

**Issue by ozturkoktay** [phizaz/diffae#74]
Hi,

I'm trying to run manipulation notebook on an A100 40 GB GPU but I'm getting out of memory error. 

Can you share your insight on this? What did you use for running manipulation process and how can I make it work? I have 4 A100s and as far as I understand the network can't be run parallel.

Thanks!

**Comment by phizaz** [comment]
You're right that the network cannot be run in parallel for manipulation tasks. I doubt that the OOM comes from the manipulation. Naive question. Have you used `torch.no_grad` during DDIM generation?
